### DISCUSSION TITLE
Server version

### DISCUSSION THREAD

**Discussion by accessdenied56** [xthursdayx/yacreaderlibrary-server-docker#61] [General]
Hi I'm pulling your image via xthursdayx/yacreaderlibrary-server-docker:latest 
How do I tell what server version is running as my app keeps telling me I have to update my server to version 9.14.1 or later

**Comment by phlegethon9** [comment]
Getting the version you can run:

`docker exec YACReaderLibraryServer YACReaderLibraryServer  -v`

if you have it up and running, otherwise you can inspect the image.  

Right now though it looks like 9.13 is the only build that is working. Below is a link to a thread @xthursdayx has open to the developers of YacReader to get some help with the new version since it has been having some issues.  

I'm running my library from an old Windows machine at the moment since it can run the latest version which will talk to the mobile app, but I'm hoping to switch back to my headless docker instance once the bugs are ironed out.

https://github.com/YACReader/yacreader/issues/417

**Reply by accessdenied56** [reply]
Thanks for the update, is this not just the 7zip version? I'm using the unarr which from what I can tell should be 9.14.1
